test/configCases/commons-chunk-plugin/hot-multi/vendor.js

```diff
@@ -0,0 +1,3 @@
+require("./common");
+
+module.exports = "vendor";
```

Comment:
Could you add the main = true check in this module too? vendor is also a main module.